### ISSUE TITLE
[CLOSED] Modualize build facility completely (e.g., tizen/gbs, ubuntu/pbuilder)

### ISSUE THREAD

**Issue by leemgs** [leemgs/TAOS-CI6#14]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Friday Jun 29, 2018 at 08:00 GMT_
_Originally opened as https://github.sec.samsung.net/STAR/TAOS-CI/issues/73_

----

We have to modualize build facility completely (e.g., tizen/gbs, ubuntu/pbuilder)

* Screenshot: 
   * https://github.sec.samsung.net/STAR/on-device-api/pull/31
![image](https://github.sec.samsung.net/storage/user/8170/files/d51ad882-7bbd-11e8-835b-106c7dc63b7c)

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/git-bot-sec"><img src="https://github.sec.samsung.net/avatars/u/63289?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [git-bot-sec](https://github.sec.samsung.net/git-bot-sec)**
_Friday Jun 29, 2018 at 08:00 GMT_

----

:octocat: **cibot**: Thank you for posting issue #73. The person in charge will reply soon.

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Friday Jun 29, 2018 at 08:07 GMT_

----

* Workaround: 

```bash
diff --git a/ci/standalone/config/config-plugins-audit.sh b/ci/standalone/config/config-plugins-audit.sh
index 5b7103c..f059df3 100644
--- a/ci/standalone/config/config-plugins-audit.sh
+++ b/ci/standalone/config/config-plugins-audit.sh
@@ -9,6 +9,7 @@ echo "[MODULE] plugins-base: Plugin group that follow Apache license with good q
 # Please append your plugin modules here.

 module_name="pr-audit-build-tizen"
+BUILD_MODE=99
 echo "[DEBUG] $module_name is started."
 echo "[DEBUG] CI/$module_name: Check if Tizen rpm package is successfully generated."
 echo "[DEBUG] Current path: $(pwd)."
@@ -17,6 +18,7 @@ $module_name
 echo "[DEBUG] $module_name is done."

 module_name="pr-audit-build-ubuntu"
+BUILD_MODE=99
 echo "[DEBUG] $module_name is started."
 echo "[DEBUG] CI/$module_name: Check if Ubuntu deb package is successfully generated."
 echo "[DEBUG] Current path: $(pwd)."
@@ -37,11 +39,11 @@ echo "[MODULE] plugins-good: Plugin group that follow Apache license with good q
 echo "[MODULE] plugins-ugly: Plugin group that does not have evaluation and aging test enough"
 # Please append your plugin modules here.

```

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Tuesday Jul 03, 2018 at 06:36 GMT_

----

PR #76 solves this issue.

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Thursday Jul 19, 2018 at 04:11 GMT_

----

Fixed by PR #125 .
